packages/forms/src/Form.js

```diff
@@ -191,4 +191,6 @@ Form.defaultProps = {
 	buttonBlockClass: 'form-actions',
 };
 
+Form.displayName = 'TalendForm';
```

Comment:
What's the problem with default value `Form`?
https://facebook.github.io/react/docs/react-component.html#displayname